### PR TITLE
 Simplify dockerfile

### PR DESCRIPTION
**Pull request by marquiz**
This change requires #128, and, it has been cherry-picked in this PR

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "intel-cmt-cat"]
+	path = intel-cmt-cat
+	url = https://github.com/01org/intel-cmt-cat.git
```
Comment (kad) [inline .gitmodules]: it is https://github.com/intel/intel-cmt-cat now.

**Comment by marquiz** [comment]
ping @balajismaniam, any review comments?

**Comment by kad** [comment]
@marquiz @okartau do we need to keep those open ? what is parth forward with it ?

**Comment by marquiz** [comment]
> do we need to keep those open ? what is parth forward with it ?

@kad: Which "those" are you referring to?

**Comment by kad** [comment]
I've seen few PRs that are quite old and still in open state, and as well, needs rebase. That was the reason of question.

**Comment by marquiz** [comment]
> I've seen few PRs that are quite old and still in open state, and as well, needs rebase. That was the reason of question.

I've been rebasing these periodically. But with no review there is no progress.

ping @balajismaniam

**Pull request closed without merging**